packages/ionic/src/commands/package/package.ts

```diff
@@ -0,0 +1,344 @@
+import {
+  CommandLineInputs,
+  CommandLineOptions,
+  LOGGER_LEVELS,
+  OptionGroup,
+  contains,
+  validators
+} from '@ionic/cli-framework';
+import { columnar } from '@ionic/cli-framework/utils/format';
+import { sleep } from '@ionic/cli-framework/utils/process';
+import chalk from 'chalk';
+import * as Debug from 'debug';
+import * as fs from 'fs';
+import * as https from 'https';
+
+import { CommandMetadata } from '../../definitions';
+import { isSuperAgentError } from '../../guards';
+import { Command } from '../../lib/command';
+import { FatalException } from '../../lib/errors';
+import { fileUtils } from '../../lib/utils/file';
+
+const debug = Debug('ionic:commands:package:build');
+const PLATFORMS = ['android', 'ios'];
+const ANDROID_BUILD_TYPES = ['debug', 'release'];
+const IOS_BUILD_TYPES = ['development', 'ad-hoc', 'app-store', 'enterprise'];
+const BUILD_TYPES = ANDROID_BUILD_TYPES.concat(IOS_BUILD_TYPES);
+
+interface PackageBuild {
+  job_id: number;
+  id: string;
+  caller_id: number;
+  platform: string;
+  build_type: string;
+  created: string;
+  finished: string;
+  state: string;
+  commit: any;
+  stack: any;
+  profile_tag: string;
+  automation_id: number;
+  environment_id: number;
+  native_config_id: number;
+  automation_name: string;
+  environment_name: string;
+  native_config_name: string;
+  job: any;
+}
+
+interface DownloadUrl {
+  url: string | null;
+}
+
+export class BuildCommand extends Command {
+  async getMetadata(): Promise<CommandMetadata> {
+    return {
+      name: 'build',
+      type: 'project',
+      summary: `Creates a package build on Appflow using name parameters, then tails the build log from Appflow and
```

Comment:
oops, sorry. Please use `description` for the long text like this. `summary` is usually a quick sentence.

You can see how it looks with `ionic package --help` and `ionic package build --help`. Please use colors to make the options stand out like in the other commands.

See `packages/ionic/src/commands/init.ts` for a summary/description example